package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
You added this one by mistake. ;)

Sending a PR to add it to `.gitignore` because it happened to me a few times as well...